### ISSUE TITLE
`check-param-names` does not work with line breaks in param doc

### ISSUE THREAD

**Issue by LukasHechenberger** [gajus/eslint-plugin-jsdoc#56]
The `check-param-names` rule does not work as expected when there are line breaks in the param doc.

Example:

```javascript
class WriteStream {
  /**
   * Writes an {@link AtviseFile} to it's corresponding node on atvise server.
   * @param {AtviseFile} file The file to write.
   * @param {function(err: Error, statusCode: node-opcua~StatusCodes, onSuccess: function)}
   * handleErrors The error handler to call. See {@link QueueStream#processChunk} for details.
   */
  processChunk(file, handleErrors) {
    ...
  }
}
```

gives the following error message:

```
/Users/.../WriteStream.js
  19:3  error  Expected @param names to be "file, handleErrors". Got "file,
handleErrors"  jsdoc/check-param-names
```

In my case the break is required because of the max line length set

**Comment by bary12** [comment]
This is a most likely a problem with [comment-parset](https://github.com/yavorskiy/comment-parser) rather than the plugin since the plugin uses it to decode the comment. However, it might be configurable to allow this.

A workaround (and a good practice) would be JSdoc's [@typedef](http://usejsdoc.org/tags-typedef.html).

**Comment by brettz9** [comment]
Will close here, but please feel free to report this at https://github.com/jsdoctypeparser/jsdoctypeparser/ , which is the library we are using that is responsible for parsing types.

If this type of type format is also supported in TypeScript, you might mention it at  https://github.com/jsdoctypeparser/jsdoctypeparser/issues/50 where missing TypeScript features are being tracked.

Thanks!
